content/news/releases/prestashop-8-2-0-available.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  PrestaShop 8.2 Is Available
+subtitle: New minor version of PrestaShop is now available
+date: 2024-09-10
+authors: [ PrestaShop ]
+image: /assets/images/2024/09/banner_8.2_1200x627.jpg
+opengraph_image: /assets/images/2024/09/banner_8.2_1200x627.jpg
+twitter_image: /assets/images/2024/09/banner_8.2_1024x512.jpg
+icon: icon-leaf
+tags:
+- version
+- minor
+- releases
+- "8.2"
+- "8.2.x"
+---
+
+It's been decided to release a new minor version of PrestaShop, 8.2. This version includes a few improvements, as well as bug fixes.
+
+![PrestaShop 8.2 is available!](/assets/images/2024/09/banner_8.2_1534x424.jpg)
+
+## What's new in PrestaShop 8.2?
+
+PrestaShop 8.2 is a minor version that includes a few improvements, as well as bug fixes, and it is relatively small compared to the previous minor versions. Here are some of the highlights.
+
+### Improvements
+
+**Correclty checking overrides during module installation**
+
+PrestaShop used to check for overrides during module installation, but the process didn’t fully block installation if a conflicting override was found. Even with a conflict and warning message, the module would still appear as installed in the database, potentially causing side effects.  This issue has been [fixed](https://github.com/PrestaShop/PrestaShop/pull/36639) and mechanism for checking overrides during the module installation has been improved.
+
+**New hook `actionFrontControllerSetVariablesBefore`**
+
+A [new hook is available](https://github.com/PrestaShop/PrestaShop/pull/36657) for assigning general purpose variables before the core does it. This hook is called `actionFrontControllerSetVariablesBefore` and is available in the `FrontController` class. You can read about potential use cases in the pull request.
+
+**Introducing of a `CartLazyArray`**
+
+[`CartLazyArray` has been introduced](https://github.com/PrestaShop/PrestaShop/pull/36575) as a first step to improve the performance of the cart object. This class is a lazy array that will only load the cart data when needed. This change introduced minor backward compatibility breaks, but it will be beneficial for the performance.
```

Comment:
```suggestion
[`CartLazyArray` has been introduced](https://github.com/PrestaShop/PrestaShop/pull/36575) as a first step to improve the cart object performance. This class is a lazy array that will only load the cart data when needed. This change introduced minor backward compatibility breaks, but it will be beneficial for the performance.
```